region/brinstar/blue.json

```diff
@@ -1140,33 +1140,38 @@
                     "h_canUsePowerBombs",
                       {"or": [
                         "canWalljump",
-                        "h_canFly",
-                          {"canShineCharge": {
-                            "usedTiles": 33,
-                            "shinesparkFrames": 43,
-                            "openEnd": 2
-                          }}
+                        "h_canFly"
                       ]}
                   ],
                   "obstacles": [
                     {
                       "id": "A",
                       "requires": [
-                        {"or": [
-                          "h_canDestroyBombWalls",
-                          {"canShineCharge": {
-                            "usedTiles": 33,
-                            "shinesparkFrames": 43,
-                            "openEnd": 2
-                          }}
-                        ]}
+                        "h_canDestroyBombWalls"
                       ]
                     }
                   ],
-                  "note": [
-                    "Base strat when entering the room from below.",
-                    "The runway here is 31 tiles before breaking the PB blocks, but becomes longer after"
-                  ]
+                  "note": "Base strat when entering the room from below."
+                },
+                {
+                  "name": "Shinespark",
+                  "notable": false,
+                  "requires": [
+                    "h_canUsePowerBombs"
+                  ],
+                  "obstacles": [
+                    {
+                      "id": "A",
+                      "requires": [
+                        {"canShineCharge": {
```

Comment:
I think we want the shinespark to be on the strat itself, and just clear the obstacle for free. As it is now, it would mean you can get back up for free if the obstacle is already cleared.